public/.eslintrc.js

```diff
@@ -18,8 +20,10 @@ module.exports = {
     'react',
   ],
   rules: {
-    'import/no-named-as-default': 0,
-    'no-console': 0,
-    'import/no-extraneous-dependencies': 0,
+    'no-return-assign': 'off',
```

Comment:
booo. This is useful, imo

Reply:
It kept complaining about my mutator functions like this one
```javascript
configureUrl: (url) => `${SERVER_CONFIGURATION.baseUrl}${url}`,
```

Would you prefer warn? or is the above an anti-pattern?

Reply:
That can't be the line of code that fails for no-return-assign...that's not an assignment at all

Reply:
Hmm. Just removed the rule disable, and there wasn't an error. It must have been some other code that no longer exists. 

But more generally speaking then, is assignment from an arrow function an antipattern?

Reply:
It's an anti-pattern to return an assignment statement, generally.

() => window.something = 'something'; is equivalent to

function() {
  return window.something = 'something';
}

would you write the above?

Reply:
... Fair point.